src/index.ts

```diff
@@ -21,18 +21,25 @@ joplin.plugins.register({
 		async function updateTodoView() {
         let notes = []
         let page_idx = 1
-        let result = (await joplin.data.get(["notes"], {fields: [
-                "id", "is_todo", "todo_due", "todo_completed", "title", "created_time"
-            ], page: page_idx}))
+        let result = (await joplin.data.get(["search"], {
+            fields: ["id", "is_todo", "todo_due", "todo_completed", "title", "created_time"],
+            query: "type:todo",
+            page: page_idx}))
         notes = notes.concat(result.items)
+
+        console.log(notes);
+
         while (result.has_more) {
             page_idx++
-            result = (await joplin.data.get(["notes"], {fields: [
-                    "id", "is_todo", "todo_due", "todo_completed", "title", "created_time"
-                ], page: page_idx}))
+            result = (await joplin.data.get(["search"], {
+                fields: ["id", "is_todo", "todo_due", "todo_completed", "title", "created_time"], 
+                query: "type:todo",
+                page: page_idx}))
             notes = notes.concat(result.items)
         }
 
+        console.log(notes);
```

Comment:
Debug code shouldn't run when not needed.

Reply:
Disregard my comment; It seems Joplin suppresses console.log output anyway, unless you run the plugin in a development mode: https://discourse.joplinapp.org/t/how-to-view-the-ouput-of-console-log-of-a-plugin-code/24771/13?u=wind0204